constants.js

```diff
@@ -17,7 +17,7 @@ const BLOCKCHAINS = {
         rpc: 'http://localhost:7545',
         hubContract: '0x209679fA3B658Cd0fC74473aF28243bfe78a9b12',
     },
-    hardhat: {
+    'hardhat:31337': {
```

Comment:
Interesting idea, however I think we should split `blockchainName` and `chainId` into separate parameters as `chainId` may be useful at some point. Or even now we can validate blockchain when connecting provider to the RPC, we can get `chainId` from the network and check if blockchain is supported. `blockchainName` is not really helpful for anything like that, it rather makes UAL more human-readable, `chainId` is a unique key that can be also received from the blockchain itself